src/main/resources/application.yml

```diff
@@ -47,6 +47,7 @@ jackson:
 
 api-url:
   prefix: /api
-ontology: ${ONTOLOGY}
+ontology: ${ONTOLOGY:unknown}
```

Comment:
I assume unknown is a real empty ontology which will cause less errors later, otherwise not defaulting to 'unknown' seems wrong.